### DISCUSSION TITLE
Custom startup splash image

### DISCUSSION THREAD

**Discussion by Herrcookiekiller** [qelectrotech/qelectrotech-source-mirror#270] [Q&A] [answered]
Where do i find https://github.com/qelectrotech/qelectrotech-source-mirror/blob/master/ico/splash.png under Windows so i can replace it with my own?

Putting splash.png in C:\Program Files\QElectroTech\ico doesnt change it.

**Comment by De-Backer** [comment, marked as answer]
Why do you want to change this?
Anyway, to do this you need to recompile the program.
After all, it is hardcoded.

**Reply by Herrcookiekiller** [reply]
Alright, fair enough. Thanks for letting me know.
